lib/portage/output.py

```diff
@@ -645,7 +645,7 @@ def ebegin(self, msg):
         msg += " ..."
         if not self.quiet:
             self.einfon(msg)
-        self.__last_e_len = len(msg) + 3
+        self.__last_e_len = len(msg) + 4
```

Comment:
... here. Why these exact values?
`" * "` has lenght 3, while `"[EE] "` etc. have length 5.

To avoid excessive length, maybe single-letter identifiers would be better (`"[E] "`, `"[W] "`, `"[Q] "` etc.)?